test/functional/healthcheck.test.js

```diff
@@ -55,13 +60,52 @@ test('proxy requests originating from behind the broker client', t => {
         });
       });
 
+      t.test('server client-health with connected client', t => {
+        request({url: serverClientHealth, json: true }, (err, res) => {
+          if (err) { return t.threw(err); }
+
+          t.equal(res.statusCode, 200, '200 statusCode');
+          t.equal(res.body['ok'], true, '{ ok: true } in body');
+          t.end();
+        });
+      });
+
+
+      t.test('server client-health after client disconnected', t => {
+        client.close();
+        setTimeout(() => {
+          request({url: serverClientHealth, json: true }, (err, res) => {
+            if (err) { return t.threw(err); }
+
+            t.equal(res.statusCode, 200, '200 statusCode');
+            t.equal(res.body['ok'], false, '{ ok: false } in body');
+            t.end();
+          });
+        }, 100);
+      });
+
+
+      t.test('server client-health after client re-connected', t => {
+        client = app.main({ port: clientPort });
+        setTimeout(() => {
+          request({url: serverClientHealth, json: true }, (err, res) => {
+            if (err) { return t.threw(err); }
+
+            t.equal(res.statusCode, 200, '200 statusCode');
+            t.equal(res.body['ok'], true, '{ ok: true } in body');
+            t.end();
+          });
+        }, 200);
+      });
+
       t.test('clean up', t => {
         client.close();
+
         setTimeout(() => {
           server.close();
           t.ok('sockets closed');
           t.end();
-        }, 100);
+        }, 300);
```

Comment:
just the 2 tests above. I wanted some more time for the to finish. This might work just as well with the old `100`